### ISSUE TITLE
Suggestion: Os

### ISSUE THREAD

**Issue by GianluPlay** [binogure-studio/city-game-studio-i18n#692]
OS as a new component, also can make your own OS to sell it to competitors or/and custom platform

**Comment by xsellier** [comment]
Just one question, what does it bring to the gameplay? What does it bring to the gameplay that custom platform doesn't?

**Comment by EngineFeaturesOfficial** [comment]
Have a custom OS make your platform unique, but elevate the cost of the platform in XX%, use a OS like vindows or lynnus doesnt elevate that much the price but there is a little royalty, the OS also give the platform a little bit more power, more power more royalty, this idea can be implemented but maybe cant make an own OS (OS isnt necessary be added to a platform, you can put no OS and you dont gonna have to pay more or less)

**Comment by GianluPlay** [comment]
> Just one question, what does it bring to the gameplay? What does it bring to the gameplay that custom paltform doesn't?

Have a custom OS make your platform unique, but elevate the cost of the platform in XX%, use a OS like vindows or lynnus doesnt elevate that much the price but there is a little royalty, the OS also give the platform a little bit more power, more power more royalty, this idea can be implemented but maybe cant make an own OS (OS isnt necessary be added to a platform, you can put no OS and you dont gonna have to pay more or less)

**Comment by xsellier** [comment]
~~Adding an OS development to City Game Studio is out of subject and won't bring anything new to the gameplay but unecessary complexity. Because to sell an OS you need to develop drivers, software applications and a lot of other stuff that is not required when you make your own video game console. There is not a single video game company that developed an OS from scratch without making its own hardware dedicated to it. That's why you cannot make an OS but a video game console.~~

Let me rephrase that:
It would be cool indeed, but it won't bring anything new to the gameplay since you can already do that when you make your own console. It is cooler to make a console since you can pick the hardware, but for the OS, it'll look like a Game Engine. So graphically it's like a game engine, but working almost as a video game console. Those are already part of City Game Studio, that's why I don't think that making an OS would bring anything new to the gameplay.

Also when you _develop_ a video game console, your team work on an OS for your console, they write drivers and everything needed for the console to work properly.

Topic is being discussed here:
https://steamcommunity.com/app/726840/discussions/0/3202622816415761656/

**Comment by xsellier** [comment]
Topic is being discussed here: https://steamcommunity.com/app/726840/discussions/0/3202622816415761656/

**Comment by xsellier** [comment]
I understand that you want to make OS in City Game Studio. And I want to make you happy guys, so I need your help to implement the feature in an elegant way. That's what I mean when I ask you what this feature bring to the gameplay.

- How to make an OS? (ie Does it require only Design point, or also Development points for example?)
- Should we be able to update an OS?
- How much developing an OS costs?
- Should we be able to sell an OS, or has to be free?
- Could we add driver implementation for specific hardware (thinking about game play, sticks and other controllers) ?
- What the incentive to make an OS?

**Comment by xsellier** [comment]
Since this issue doesn't fit the issue template, nor explains how or why it enhance the gameplay, I'm closing it.
